[2025-02-02T16:09:07.890+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-02T16:09:07.901+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_hooks.featur_engineer manual__2025-02-02T16:08:48.446440+00:00 [queued]>
[2025-02-02T16:09:07.905+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_hooks.featur_engineer manual__2025-02-02T16:08:48.446440+00:00 [queued]>
[2025-02-02T16:09:07.905+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-02-02T16:09:07.910+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): featur_engineer> on 2025-02-02 16:08:48.446440+00:00
[2025-02-02T16:09:07.916+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=421) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-02T16:09:07.916+0000] {standard_task_runner.py:64} INFO - Started process 428 to run task
[2025-02-02T16:09:07.917+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks', 'featur_engineer', 'manual__2025-02-02T16:08:48.446440+00:00', '--job-id', '1483', '--raw', '--subdir', 'DAGS_FOLDER/Stock_pred.py', '--cfg-path', '/tmp/tmpwx6pslgz']
[2025-02-02T16:09:07.918+0000] {standard_task_runner.py:91} INFO - Job 1483: Subtask featur_engineer
[2025-02-02T16:09:07.935+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_postgres_hooks.featur_engineer manual__2025-02-02T16:08:48.446440+00:00 [running]> on host b7aba691e1ff
[2025-02-02T16:09:07.969+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_postgres_hooks' AIRFLOW_CTX_TASK_ID='featur_engineer' AIRFLOW_CTX_EXECUTION_DATE='2025-02-02T16:08:48.446440+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-02T16:08:48.446440+00:00'
[2025-02-02T16:09:07.970+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-02T16:09:08.279+0000] {warnings.py:112} WARNING - /opt/***/dags/Stock_pred.py:237: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  input.drop(columns=["Target","CLOSE_t+1"],inplace=True)

[2025-02-02T16:09:08.284+0000] {warnings.py:112} WARNING - /opt/***/dags/Stock_pred.py:240: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  input['CLOSE_diff1'] = input['CLOSE'].diff(1)

[2025-02-02T16:09:08.290+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-02-02T16:09:08.290+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-02T16:09:08.293+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks, task_id=featur_engineer, run_id=manual__2025-02-02T16:08:48.446440+00:00, execution_date=20250202T160848, start_date=20250202T160907, end_date=20250202T160908
[2025-02-02T16:09:08.304+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-02-02T16:09:08.318+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-02T16:09:08.318+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
